cmd/kube-apiserver/app/aggregator.go

```diff
@@ -79,10 +79,11 @@ func createAggregatorConfig(kubeAPIServerConfig genericapiserver.Config, command
 		}
 	}
 	aggregatorConfig := &aggregatorapiserver.Config{
-		GenericConfig:       &genericConfig,
-		CoreAPIServerClient: client,
-		ProxyClientCert:     certBytes,
-		ProxyClientKey:      keyBytes,
+		GenericConfig:           &genericConfig,
+		CoreAPIServerClient:     client,
+		ProxyClientCert:         certBytes,
+		ProxyClientKey:          keyBytes,
+		EnableAggregatorRouting: commandOptions.EnableAggregatorRouting,
```

Comment:
and the interface would be initialized here if `commandOptions.EnableAggregatorRouting` was true